lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_lint_localizations.rb

```diff
@@ -22,17 +32,38 @@ def self.run_linter(params)
           install_path: resolve_path(params[:install_path]),
           version: params[:version]
         )
-        all_violations = helper.run(
+
+        helper.run(
           input_dir: resolve_path(params[:input_dir]),
           base_lang: params[:base_lang],
           only_langs: params[:only_langs]
         )
+      end
+
+      def self.report(violations:, base_lang:)
+        violations.each do |lang, lang_violations|
+          UI.error "Inconsistencies found between '#{base_lang}' and '#{lang}':\n\n#{lang_violations.join("\n")}\n"
+        end
+      end
+
+      def self.find_duplicated_keys(params)
+        duplicate_keys = {}
 
-        all_violations.each do |lang, lang_violations|
-          UI.error "Inconsistencies found between '#{params[:base_lang]}' and '#{lang}':\n\n#{lang_violations.join("\n")}\n"
+        files_to_lint = Dir.chdir(params[:input_dir]) do
+          Dir.glob('*.lproj/Localizable.strings').map do |file|
+            {
+              language: File.basename(File.dirname(file), '.lproj'),
+              path: File.join(params[:input_dir], file)
+            }
+          end
         end
```

Comment:
Since we're already in `chdir(input_dir)`, is it really necessary to `File.join(input_dir, file)` to pass to the `path:` parameter?

Alternatively, since you `File.join` anyway, instead of using `Dirchdir` you could use the `base:` param of `Dir.glob` (which returns relative paths to the `base` but since you `File.join` anyway…

```suggestion
        files_to_lint = Dir.glob('*.lproj/Localizable.strings', base: params[:input_dir]).map do |file|
          {
            language: File.basename(File.dirname(file), '.lproj'),
            path: File.join(params[:input_dir], file)
          }
        end
```